.github/workflows/build.yml

```diff
@@ -182,5 +182,5 @@ jobs:
         submodules: true
     - run: sudo apt-get install ninja-build
     - run: make clang-debug
-    - name: tests (excluding memory64)
-      run: ./test/run-tests.py --exclude-dir memory64
+    - name: tests (wasm2c tests excluding memory64)
+      run: ./test/run-tests.py *wasm2c* --exclude-dir memory64
```

Comment:
I don't *think* it needs the stars here (which might get interpreted by the shell) -- I think just `./test/run-tests.py wasm2c --exclude-dir memory64` is enough.